website_authorized_service_centers/views/authorized_service_centers.xml

```diff
@@ -16,7 +16,7 @@
                     </section>
                     <section class="mt16 mb16">
                         <div class="container">
-                            <t t-set="providers" t-value="env['product.brand'].search([('authorized_partner_ids', '!=', False)], order='name desc')"/>
+                            <t t-set="providers" t-value="env['product.brand'].sudo().search([('authorized_partner_ids', '!=', False)], order='name desc')"/>
```

Comment:
@moylop260 the ACL that you suggest us, already exists:
https://github.com/Vauxoo/product-attribute/blob/8.0/product_brand/security/ir.model.access.csv#L3

The ACL requiered is for the model res.partner, because the info aren't returned and rendered correctly caused by the lack of the permissions over the res.partner model.

I like the use of the stament `sudo()` in the template instead of ACL soluction because it's not adding an extra granting permissions also , the `sudo()` stament only works in backend, returning pure html after qweb process.